packages/hash/api/src/collab/save.ts

```diff
@@ -80,10 +82,8 @@ const ensureEntityTypeForComponent = async (
   }
 
   if (!desiredEntityTypeId) {
-    const componentMeta = await fetchBlockMeta(componentId);
-    const jsonSchema = JSON.parse(
-      JSON.stringify(componentMeta.componentSchema),
-    );
+    const block = await fetchBlock(componentId);
+    const jsonSchema = JSON.parse(JSON.stringify(block.schema));
```

Comment:
I understand this was not introduced in this PR, but why do we stringify and parse here?

Reply:
It's a crude way of cloning it, because I mutate it below to remove `editableRef` from the schema if its there in order to ensure it doesn't end up in the relevant entity type.